storageprovider/storage_provider.go

```diff
@@ -10,12 +10,14 @@ import (
 )
 
 const (
-	arrayIPKey     = "arrayIp"
+	arrayIPKey     = "backend"
 	usernameKey    = "username"
 	passwordKey    = "password"
-	portKey        = "port"
+	portKey        = "servicePort"
```

Comment:
Can you change the key names too?

Reply:
updated other places too to use Backend instead of ArrayIP and ServicePort instead of Port. Added default port as well.